### DISCUSSION TITLE
Acount Erstellen/register

### DISCUSSION THREAD

**Discussion by codajoda** [SE-TINF22B2/G3-ApeRepublic#5] [User-Stories]
Als Benutzer will ich einen Account bei ApeRepublic mit miener Email und Password erstellen. Dann bekomme ich eine Bestätigung per Email.

**Comment by Marc738** [comment]
Potenzielle Issues:
* Wie werden E-Mail und Passwort in der Datenbank gespeichert?
* Wie wird die Bestätigungsmail versand?
* Was ist wenn das Passwort vergessen wurde?

**Reply by MatejEvc** [reply]
Brauchen wir eine E-Mail? Wäre das zu kompliziert mit Bestätigungsmail?

**Reply by elnaz-gharoon** [reply]
Die Bestätigung könnte auch durch eine Nachrichtmeldung in App gezeit werden, nicht unbedingt per Email
